src/serac/physics/state/finite_element_vector.cpp

```diff
@@ -15,9 +15,6 @@ FiniteElementVector::FiniteElementVector(const mfem::ParFiniteElementSpace& spac
       space_(std::make_unique<mfem::ParFiniteElementSpace>(space, &mesh_.get(), coll_.get())),
       name_(name)
 {
```

Comment:
Are we sure we want to remove error checking?

Reply:
I'm definitely not a fan of removing it. This was mostly temporary. I think we will have issues with tribol if we re-enable it.

Reply:
Can you make an issue to track hardening this change? We should put as many useful errors and checks as we can but it can be a follow up PR.

Reply:
Added, and I added back a limited assertion which only checks ordering for vector fields, since the two orderings are identical for vdim=1. This still allows the tests to pass (although tribol may still be messing up the order when calculating the forces).